cmd/root_test.go

```
@@ -54,60 +54,68 @@ func TestRootCmdWithCron(t *testing.T) {
-	cmd.Execute()
 
-	out, err := io.ReadAll(b)
-	if err != nil {
-		t.Fatal(err)
-	}
+	t.Run("it should return error when job file can not be opened", func(t *testing.T) {
+		cmd := rootCmd
+		cmd.SetArgs([]string{"-f", "/Users/jobs.yaml"})
+		b := bytes.NewBufferString("")
+		cmd.SetErr(b)
+		cmd.Execute()
```

golangci-lint (1.64.8)

Error return value of `cmd.Execute` is not checked

(errcheck)